### PR TITLE
Update code snippets using black formatter.

### PR DESCRIPTION
**Pull request by abhishekchitwar**
@RobPasMue @Revathyvenugopal162 
Update code snippets using black formatter.

### CODE REVIEW

File: cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex [hunk 1]
```diff
@@ -44,8 +44,9 @@ \section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Launch Fluent l
 This code starts Fluent in the background so that commands can be sent to Fluent from the Python interpreter.
 
 \begin{lstlisting}[language=Python]
-import ansys.fluent.core as pyfluent
-solver = pyfluent.launch_fluent(mode = "solver", show_gui = True)
+  import ansys.fluent.core as pyfluent
```
Comment (Revathyvenugopal162) [inline cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex:47]:
```suggestion
import ansys.fluent.core as pyfluent
```
Do we require this indent before? is it a suggestion from review?

Comment (RobPasMue) [reply cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex:47]: That indendt shouldn't be there

**Comment by RobPasMue** [comment]
Why is there indents throughout the entire LaTeX file?

**Comment by abhishekchitwar** [comment]
yeah, if not recommended will remove those indents

File: cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex (hunk 1 above)
Comment (abhishekchitwar) [reply cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex]: Removed all the indent!

**Comment by abhishekchitwar** [comment]
Removed all the indents!

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**